### ISSUE TITLE
Add Sample data for development

### ISSUE THREAD

**Issue by erikwj** [lunatech-labs/lunatech-competence-center-online#56]
Create an SQL script that loads a list of sample data into the postgres db

**Comment by ag91** [comment]
Cannot we copy the contents of the database we currently have?
For instance, [this question](https://stackoverflow.com/questions/11407349/mysql-how-to-export-and-import-an-sql-file-from-command-line) is relevant for MySQL.

**Comment by erikwj** [comment]
Yes, good idea, we could probably dump some data from "production" with pg_dump

**Comment by eamelink** [comment]
@GoldenZero what's the status of this?
